### PR TITLE
fix Line Guns

### LINKED ISSUE #1150: Line Guns have a single tile worth of "dead space".
Description of issue: Line guns have a tile of dead space (hah) between them and the actual projectile, where people are safe.

Difference between expected and actual behavior: You shouldn't be in melee range with someone and be safe when they shoot a line gun at you. But that is the case. There's a tile of safe space before the line gun's projectiles appear, and I've seen it used in-game in firing lines purposefully, even though it's a bug.

Steps to reproduce: Stand face to face with someone and shoot past them with a line gun. It won't hurt them.

### PR DESCRIPTION
**Pull request by Evankhell561**
fix: #1150
:cl:
fix: Line Guns have a single tile worth of "dead space". 
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ds13/line_cutter.dm [hunk 1]
```diff
@@ -14,7 +14,6 @@
 	icon_state = "linecutter"
 	item_state = "linecutter"
 
-	ammo_type = /obj/item/ammo_casing/linerack
```
Comment (JakeDaBoss) [inline code/modules/projectiles/guns/ds13/line_cutter.dm:17]: This seems to be removed. Why is that?

Comment (Evankhell561) [reply code/modules/projectiles/guns/ds13/line_cutter.dm:17]: duplicate var:
image

Comment (Evankhell561) [reply code/modules/projectiles/guns/ds13/line_cutter.dm:17]: image

Comment (Evankhell561) [reply code/modules/projectiles/guns/ds13/line_cutter.dm:17]: even so, I had forgotten this change, I did it when I was trying to apply this fix in a different way.

**Review by JakeDaBoss — APPROVED** [review]

**Review by NanakoAC — APPROVED** [review]
sorry for delay, wanted to test this first, done so

**Pull request merged**